scripts/common/preflights.sh

```diff
@@ -30,7 +30,7 @@ function require64Bit() {
 
 function bailIfUnsupportedOS() {
     case "$LSB_DIST$DIST_VERSION" in
-        ubuntu16.04|ubuntu18.04|rhel7.4|rhel7.5|rhel7.6|centos7.4|centos7.5|centos7.6)
+        ubuntu16.04|ubuntu18.04|rhel7.4|rhel7.5|rhel7.6|rhel7.7|centos7.4|centos7.5|centos7.6|centos7.7)
```

Comment:
The images we use to download and bundle the k8s and docker packages are `centos:7`, which has been `7.7` for the past 3 months. The only reason 7.7 install hasn't been working is because of this check. What about changing this to check for 7.4+ so that when 7.8 is released we support that automatically? Probably a product question.